frontend/components/ResourceCounts.tsx

```diff
@@ -0,0 +1,67 @@
+import { useEffect, useState } from 'react';
+import { Badge, Button, Stack } from '@mantine/core';
+import Link from 'next/link';
+
+/**
+ * Component which retrieves all resources and their counts and translates them into buttons
+ * @returns array of JSX Buttons
+ */
+const ResourceCounts = () => {
+  // set initial state to no resources
+  const [resources, setResources] = useState<Record<string, number>>({
+    Measure: 0,
+    Library: 0
+  });
+  useEffect(() => {
+    // Note: count may not update without refresh if there ends up being a way to change it on the server through the app
+    Promise.all([
+      fetch(`${process.env.NEXT_PUBLIC_MRS_SERVER}/Measure`),
+      fetch(`${process.env.NEXT_PUBLIC_MRS_SERVER}/Library`)
+    ])
+      .then(([resMeasure, resLibrary]) => Promise.all([resMeasure.json(), resLibrary.json()]))
+      .then(([measureBundle, libraryBundle]) => {
+        setResources({ Measure: measureBundle.total || 0, Library: libraryBundle.total || 0 });
+      })
+      .catch(error => {
```

Comment:
I also noticed this (I was using an incompatible node version and so I never connected to the mongo database and was wondering why there were no resources loaded 🤦‍♀️ lol) I think adding the Mantine notification is a good idea and probably won't be a lot of overhead to implement since we already have it implemented in the deqm-test-server-frontend

Reply:
I have an initial take on this, which shows the error in the resources section instead of showing the 0 count resources. I'm mixed on it as an approach and would be interested in opinions.

Reply:
Added a bit of styling. Basic look:
<img width="1377" alt="Screen Shot 2023-04-16 at 2 28 12 PM" src="https://user-images.githubusercontent.com/2643955/232333925-a2a61c86-19bd-454f-b238-6e384ec49e7e.png">